src/ui-kit/components/Layouts/Header/Header.jsx

```diff
@@ -0,0 +1,37 @@
+import { Typography } from '@mui/material';
+import {
+  HeaderWrap,
+  Logo,
+  NotificationCount,
+  NotificationIcon,
+  NotificationWrap,
+  AuthSection,
+  StyledSpan,
+  UserName,
+  HeaderToogle,
+} from './Header.styled';
+
+const Header = () => {
```

Comment:
You do not need requests to change the theme, you can immediately make the theme change functionality